hera_sim/tests/test_vis.py

```diff
@@ -5,7 +5,11 @@
 
 from hera_sim import visibilities as vis, io
```

Comment:
I'm not sure if I'm missing something, but why would you give a single module two aliases?

Reply:
That's funny, because it's exactly how I thought the syntax there worked, but it actually is equivalent to:
```python
from hera_sim import visibilities as vis
from hera_sim import io
```

Reply:
I see; that actually makes sense. That said, I think that the equivalent statements (what you posted in a code block) is preferable, even if both implementations do the same thing. The two import statements are more readable than a single import statement.

Reply:
Ok, made the change.